bundles/org.openhab.binding.fenecon/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -174,5 +208,61 @@
 		<category>Energy</category>
 		<state readOnly="true" pattern="%.0f %unit%"/>
 	</channel-type>
+	<channel-type id="inverter-air-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Inverter Air Temperature</label>
+		<description>Air temperature at the inverter in °C</description>
```

Comment:
Better not mention the unit in the description, as the unit depends on the user set locale. Also applies to the other channels.